test/visualTest.ts

```diff
@@ -117,6 +117,16 @@ module powerbi.extensibility.visual.test {
                 });
             });
 
+            it("Tasks Elements is present in dom, if only available task name", (done) => {
```

Comment:
`Task Elements are presented in DOM if and only if task name is available (specified)`

Reply:
Done